### PR TITLE
Update to latest hipe4ml and add binary case

### LINKED ISSUE #109: Move to newest release of hipe4ml
Adapt analysis code to newest developments in [hipe4ml v0.0.6](https://github.com/hipe4ml/hipe4ml/tree/v0.0.6)

### PR DESCRIPTION
**Pull request by fcatalan92**
To perform the binary classification, the FD file in the configuration must be set to null and the label lists must have only two entries.
Fix #109

### CODE REVIEW

File: ML/MLClassification.py [hunk 1]
```diff
@@ -272,33 +297,47 @@ def main():
         inputCfg = yaml.load(ymlCfgFile, yaml.FullLoader)
     print('Loading analysis configuration: Done!')
 
-    print('Loading data files: ...', end='\r')
-    PromptDf = LoadDfFromRootOrParquet(inputCfg['input']['prompt'])
-    FDDf = LoadDfFromRootOrParquet(inputCfg['input']['FD'])
-    DataDf = LoadDfFromRootOrParquet(inputCfg['input']['data'])
-    print('Loading data files: Done!')
+    print('Loading and preparing data files: ...', end='\r')
+    PromptHandler = TreeHandler(inputCfg['input']['prompt'])
```
Comment (fgrosa) [inline ML/MLClassification.py:301]: Does the `TreeHandler` support also a list of files as input?

Comment (fcatalan92) [reply ML/MLClassification.py:301]: Not at the moment. We could add this possibility in hipe4ml

**Comment by fcatalan92** [comment]
> Thanks a lot @fcatalan92! Even if it is not relevant, but for consistency could you also change the hipe4ml version in the README?

Done

File: ML/MLClassification.py (hunk 1 above)
Comment (fgrosa) [reply ML/MLClassification.py:301]: Yes indeed I agree! The feature would be useful to be able to pass more than one file (i.e. to use prompt+FD as signal or merge different pT bins). Let me know if I can help you with hipe4ml!

**Pull request merged**